ai_flow/frontend/src/views/metadata/Artifact.vue

```diff
@@ -55,13 +55,16 @@ limitations under the License. -->
         </span>
       </s-table>
     </a-card>
+    <a-card :bordered="false">
```

Comment:
I am not very familiar with Vue so correct me if I am wrong. I am wondering if we can have a common place to add the version to the view instead of copying the code in many places.

Reply:
@Sxnan, this pull request didn't make a common place for version for the time being. It will be optimized uniformly in the future.